trade_remedies_public/cases/v2_views/invite.py

```diff
@@ -203,9 +203,11 @@ class ReviewInvitation(BaseInviteView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["cases"] = self.client.organisations(
-            self.request.user.contact["organisation"]["id"], fields=["cases"]
+        cases = self.client.organisations(
+            self.request.user.contact["organisation"]["id"], fields=["cases", "user_cases"]
         )
+        context["cases"] = cases
```

Comment:
cases is currently an OrganisationObject, you will need to access the specific 'cases' property of that object:

```suggestion
        context["cases"] = cases.cases
```

That being said, it's probably best to rename 'cases' to something more readable, like 'organisation'

Reply:
Resolved in https://github.com/uktrade/trade-remedies-public/pull/255/commits/4fe509f67bcb959c2be68b341ccec99565fb5059.